app/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -409,6 +412,14 @@ const groupPage = () => (
       <EntityPageManageProgrammeAdminContent />
     </EntityLayout.Route>
 
+    <EntityLayout.Route
+      path="/manage-members"
+      title="Manage Members"
+      if={isEntityWith({ kind: 'group', type: 'delivery-project' })}
+    >
+      <EntityPageManageProjectUserContent />
+    </EntityLayout.Route>
+
```

Comment:
Tests for this new route - e.g. https://github.com/DEFRA/adp-portal/blob/aa0a83a71957522978d2de1e422e1f99af5d0295/app/packages/app/src/components/catalog/EntityPage.test.tsx#L378-L380